src/PortingAssistantExtensionServer/Handlers/PortingAssistantTextSyncHandler.cs

```diff
@@ -140,7 +140,7 @@ private async void Process(CodeFileDocument document)
             }
             catch (Exception ex)
             {
-                _logger.LogError("Incremental assessment failed with error:", ex);
+                _logger.LogError("Incremental assessment failed with error: {0}", ex);
```

Comment:
You can change this to use the signature where you pass the exception itself. This will make sure we get the full exception. I think it's LogError(ex, "Message")